strings/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Please revert the changes here.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable